### PR TITLE
Added BlackoilWellModel::getLocalNonshutWells to reduce code duplicat…

### PR DESCRIPTION
**Pull request by blattms**
…ion.

Some drive by code duplication reduction.

### CODE REVIEW

**Comment by blattms** [comment]
the sorting should not be there. Will remove.

**Comment by blattms** [comment]
gone.

**Comment by blattms** [comment]
jenkins build this please

File: opm/simulators/wells/BlackoilWellModel_impl.hpp [hunk 1]
```diff
@@ -214,7 +214,16 @@ namespace Opm {
     }
 
 
-
+    template<typename TypeTag>
+    std::vector< Well >
+    BlackoilWellModel<TypeTag>::
+    getLocalNonshutWells(const int timeStepIdx, int& globalNumWells) const
```
[the remark this answers is not in the document]
Comment (atgeirr) [reply opm/simulators/wells/BlackoilWellModel_impl.hpp:220]: While this is also nice code, the existing code is nice enough. I'll merge this now. Not 100% sure I think std::tie is so nice we should use it always, maybe if we get some more MATLAB-ish syntactic sugar for it?

**Comment by blattms** [comment]
Thanks for the quick merge. I was torn about using pair and tie or not before pushing myself.

**Comment by atgeirr** [comment]
> I was torn about using pair and tie or not before pushing myself.

If you come out in favour of tie(), I'll merge that too...

**Pull request merged**